### PR TITLE
Added SuperAdmin Dashboard link to FIND GovUKHeader.

### PR DESCRIPTION
**Pull request by paul-lawlor-tco**
## Description

GAP 2043: https://technologyprogramme.atlassian.net/browse/GAP-2043 

Summary of the changes and the related issue. List any dependencies that are required for this change:

Adds a link to the Super Admin Dashboard from the FIND header (if the user is an authenticated Super Admin).

THERE IS AN INFRA PR TO ADD THE NEW ENV VAR (ADMIN_HOST) TO THE APP: https://eu-west-2.console.aws.amazon.com/codesuite/codecommit/repositories/find-a-grant-infra/pull-requests/272/changes?region=eu-west-2

## Type of change

Please check the relevant options.

- [ ] Bug fix (non-breaking change which fixes an issue).
- [X] New feature (non-breaking change which adds functionality).
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected).
- [ ] This change requires a documentation update.

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes:

- [x] Unit Test

- [ ] Integration Test (if applicable)

- [ ] End to End Test (if applicable)

## Screenshots (if appropriate):

Please attach screenshots of the change if it is a UI change:
Screenshot 2024-02-15 at 14 53 54

# Checklist:

- [ ] If I have listed dependencies above, I have ensured that they are present in the target branch.
- [x] I have performed a self-review of my code.
- [ ] I have commented my code in hard-to-understand areas.
- [ ] I have made corresponding changes to the documentation where applicable.
- [ ] I have ran cypress tests and they all pass locally.

### CODE REVIEW

File: .env.example [hunk 1]
```diff
@@ -29,6 +29,7 @@ NEW_MANDATORY_QUESTION_JOURNEY_ENABLED=newMandatoryQuestionJourneyEnabled
 NOTIFICATION_UNSUBSCRIBE_JWT_EXPIRY_TIME=7d
 VERIFICATION_JWT_EXPIRY_TIME=7d
 APPLICANT_HOST=http://localhost:3001/apply/applicant
+ADMIN_HOST=http://localhost:3000/apply/admin
```
Comment (john-tco) [inline .env.example:32]: Infra pr for this var is in the ticket - needs to be merged before this

Comment (paul-lawlor-tco) [reply .env.example:32]: PR: https://eu-west-2.console.aws.amazon.com/codesuite/codecommit/repositories/find-a-grant-infra/pull-requests/272/changes?region=eu-west-2

Comment (ryan-tco) [reply .env.example:32]: Might be worth linking this in one of the channels for eyes

Comment (paul-lawlor-tco) [reply .env.example:32]: Nice! Got it linked in the Jupiter channel

**Review by ryan-tco — APPROVED** [review]

**Review by dylanwrightCO — APPROVED** [review]

**Pull request merged**